src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -722,15 +722,26 @@ export class OnlineGameWrapperComponent extends GameWrapper implements OnInit, O
             }
         }
     }
-    public getPlayerNameFontColor(player: number): { [key: string]: string} {
-        if (this.observerRole === player || this.observerRole > 1) {
-            return { color: 'black' };
-        } else if (this.opponentIsOffline()) {
-            return this.OFFLINE_FONT_COLOR;
+    public getPlayerNameClass(player: number): string {
+        if (this.opponentIsOffline()) {
+            return 'is-grey-light';
         } else {
-            return { color: 'black' };
+            if (player === 0) {
+                return 'is-white';
+            } else {
+                return 'is-black';
+            }
         }
     }
+    public getBoardHighlight(): string[] {
+        if (this.observerRole != null && this.isCurrentPlayer()) {
+            return ['player' + this.getPlayer().value + '-bg'];
+        }
+        return [];
+    }
+    private isCurrentPlayer(): boolean {
```

Comment:
isUserCurrentPlayer ? et lire le code confirme que this.observerRole != null est inutile non?

Reply:
tiens du coup oui comme null != un nombre

Reply:
(fait)